tests/Rest/ChannelTest.php

```diff
@@ -425,7 +425,7 @@ public function httpBindingsProvider(): array
                     'array' => true
                 ],
             ],
-            'Get thread members' => [
+            'Get thread members 2' => [
```

Comment:
This was just an incorrect copy paste, can be removed instead

Reply:
That was my first thought, kept it so you could make the decision of what should happen to it.